lib/src/components/NxTag/NxTag.scss

```diff
@@ -0,0 +1,170 @@
+/**
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+ @import '../../scss-shared/nx-variables';
+ @import '../../scss-shared/nx-text-helpers';
+ @import '../../scss-shared/nx-container-helpers';
+
+.nx-tag {
+  @include semi-bold();
+  align-items: center;
+  background-color: $nx-blue-800;
+  border: 1px solid $nx-blue-800;
+  border-radius: 16px;
+  box-sizing: border-box;
+  color: $nx-text-color-dark;
+  cursor: default;
+  display: inline-block;
+  font-size: $nx-font-size-s;
+  margin: 0 $nx-spacing-xs $nx-spacing-xs 0;
+  padding: 5px $nx-spacing-xs 6px $nx-spacing-md;
+  max-width: 824px;
+}
+
+.nx-tag__text {
+  @include nx-truncate-ellipsis;
+  padding-right: $nx-spacing-xs;
+}
+
+.nx-tag--selectable {
+  cursor: pointer;
+  display: inline-flex;
+
+  &:focus {
+    border: 1px solid $nx-focus-border-color;
+    box-shadow: $nx-focus-box-shadow;
+    outline: 0;
+  }
+
+  &:hover {
+    border-color: $nx-grey-900;
+  }
+}
+
+.nx-tag--selected {
+  background-color: $nx-blue-800;
+  border-color: $nx-blue-800;
+  color: $nx-text-color-dark;
+
+  &:focus {
+    border: 1px solid $nx-focus-border-color;
+  }
+
+  .nx-tag__action {
+    .nx-icon {
+      transform: rotate(-45deg);
+    }
+  }
+}
+
+.nx-tag--unselected {
+  border-color: $nx-grey-300;
+}
+
+.nx-tag__action {
+  @include container-horizontal;
+  margin-left: $nx-spacing-xs;
+}
+
+// Available tag colours
+.nx-tag--default {
+  color: $nx-white;
+
+  &.nx-tag--unselected {
+    background-color: $nx-grey-300;
+    color: $nx-text-color-dark;
+  }
+}
+
+.nx-tag--light-blue {
+  background-color: $nx-blue-200;
+  border-color: $nx-blue-200;
+
+  &.nx-tag--unselected {
+    background-color: $nx-grey-300;
+  }
+}
+
+.nx-tag--purple {
+  background-color: #d584ff;
+  border-color: #d584ff;
+
+  &.nx-tag--unselected {
+    background-color:#f3dcff;
+  }
+}
+
+.nx-tag--pink {
+  background-color: #ff90d1;
```

Comment:
The swatch variables have been merged to master, so you should probably merge into this branch and update these with the variables.

Reply:
> The swatch variables have been merged to master, so you should probably merge into this branch and update these with the variables.

Done as part of the mixin: https://github.com/sonatype/sonatype-react-shared-components/pull/290/commits/1d3cc026cbc0bb889f3d262babddf46b9ba17dba